src/main/java/uk/gov/justice/digital/service/operationaldatastore/OperationalDataStoreTransformation.java

```diff
@@ -30,9 +27,7 @@ Dataset<Row> transform(Dataset<Row> dataFrame) {
         // We normalise columns to lower case to avoid having to quote every column due to Postgres lower casing everything in incoming queries
         Dataset<Row> lowerCaseColsDf = normaliseColumnsToLowerCase(dataFrame);
         // Handle 0x00 null String character which cannot be inserted in to a Postgres text column
-        Dataset<Row> withoutNullsDf = stripNullStrings(lowerCaseColsDf);
-        // We don't store these metadata columns in the destination table so we remove them
-        return withoutNullsDf.drop(OPERATION.toLowerCase(), TIMESTAMP.toLowerCase(), CHECKPOINT_COL.toLowerCase());
```

Comment:
Most of the transformation is common between batch and CDC, but the columns are not (since CDC needs the op column in the temporary table for the merge)